src/main/java/com/linecorp/armeria/server/http/tomcat/TomcatServiceBuilder.java

```diff
@@ -44,6 +51,10 @@
 
     private static final Logger logger = LoggerFactory.getLogger(TomcatServiceBuilder.class);
 
+    // From Tomcat conf/server.xml
+    private static final String DEFAULT_SERVICE_NAME = "Catalina";
+    private static final String DEFAULT_ENGINE_NAME = DEFAULT_SERVICE_NAME;
```

Comment:
Sorry, I missed PR note. checked and confirmed `conf/server.xml` use this name for both.